src/main/java/com/google/ehub/servlets/BlobstoreUploadServlet.java

```diff
@@ -0,0 +1,33 @@
+package com.google.ehub.servlets;
+
+import com.google.appengine.api.blobstore.BlobstoreService;
+import com.google.appengine.api.blobstore.BlobstoreServiceFactory;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Handles GET requests for upload URL from Blobstore API
+ */
+@WebServlet("/blobstore-upload")
+public class BlobstoreUploadServlet extends HttpServlet {
```

Comment:
Are you planning to let users upload images or are you going to keep a copy that you get from the API?

Reply:
I am planning to store images from the API in our own Blobstore. Also this Servlet can be useful for the Profile page in case we make users upload profile pictures.